src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -498,6 +498,16 @@ CustomTextLayout::CustomTextLayout(IDWriteFactory2* const factory,
             glyphRunDescription.stringLength = run.textLength;
             glyphRunDescription.textPosition = run.textStart;
 
+            // Calculate the position *after* drawing, this will be used if we're in RTL.
+            // Original comment:

```

Comment:
Don't leave text saying "original comment". Just reconcile the new behavior and leave appropriate comments throughout.